packages/workers-shared/asset-server-worker/src/responses.ts

```diff
@@ -0,0 +1,52 @@
+export class OkResponse extends Response {
+	constructor(body: BodyInit | null, init?: ResponseInit) {
+		super(body, {
+			...init,
+			status: 200,
+		});
+	}
+}
+
+export class NotFoundResponse extends Response {
+	constructor(...[body, init]: ConstructorParameters<typeof Response>) {
+		super(body, {
+			...init,
+			status: 404,
+			statusText: "Not Found",
+		});
+	}
+}
+
+export class MethodNotAllowedResponse extends Response {
+	constructor(...[body, init]: ConstructorParameters<typeof Response>) {
+		super(body, {
+			...init,
+			status: 405,
+			statusText: "Method Not Allowed",
+		});
+	}
+}
+
+export class InternalServerErrorResponse extends Response {
+	constructor(err: Error, init?: ResponseInit) {
+		let body: string | undefined = undefined;
+		// eslint-disable-next-line @typescript-eslint/no-explicit-any
+		if ((globalThis as any).DEBUG) {
+			body = `${err.message}\n\n${err.stack}`;
+		}
```

Comment:
How would this DEBUG be set? Via an esbuild `define`?
Would this appear in production?

Reply:
ah yes, good catch. Removing for now